cmd/stargz-fuse-manager/fusemanager/api/api.proto

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
> Can we have `fusemanager` package in the repo root?
I have tried two approaches:

1. Placing the fusemanager in the repository root. However, since the fsopts package is located in the cmd directory, building directly leads to issues where fusemanager cannot find fsopts. This requires adding fsopts separately to the go.mod file, which would introduce a significant number of dependencies.

2. Keeping the service.go file that depends on fsopts, along with the fusemanager.go and fusestore.go files, in the cmd/stargz-fuse-manager directory, while only moving the client-side content to the repository root.

Do you have any better solutions?

Reply:
Can we pass `fsopts.ConfigFsOpts` to fusemanager as a callback function so that we can eliminate direct dependency from fusemanager to fsopts?

Reply:
done